modules/artifact/src/main/java/org/mule/runtime/module/artifact/classloader/MuleArtifactClassLoader.java

```diff
@@ -109,21 +135,63 @@ public void setResourceReleaserClassLocation(String resourceReleaserClassLocatio
     this.resourceReleaserClassLocation = resourceReleaserClassLocation;
   }
 
-  protected ResourceReleaser createResourceReleaserInstance() {
+  public void setErrorHooksClassLocation(String errorHooksClassLocation) {
```

Comment:
This is how resource releaser functionality was tested, I didn't find a way to do it without allowing to override the method (it could be protected instead of public).